### ISSUE TITLE
Possible incompatibility with Oculus/Iris in combination with Drippy [Could be related to Bad Access issue]

### ISSUE THREAD

**Issue by Keksuccino** [Keksuccino/FancyMenu#535]
- https://discord.com/channels/746096190592122911/912100384410062848/1088617746817884221
- Issue in Drippy repo: https://github.com/Keksuccino/Drippy-Loading-Screen/issues/47
- Oculus (Forge) on latest version seems to trigger the Bad Access (#506) issue when Drippy is installed
- Same happens with Iris (Fabric)

**Comment by MattB70** [comment]
Here's another crash log if it helps.

log: https://gist.github.com/MattB70/cb015308c934cf3008e53de5d93b5ee4#file-crash-txt

**Comment by Keksuccino** [comment]
> Here's another crash log if it helps.
> 
> log: https://gist.github.com/MattB70/cb015308c934cf3008e53de5d93b5ee4#file-crash-txt

Hi! When exactly happend this? On game launch in the loading screen? Or later while in the layout editor?
Do you have Oculus or Iris installed?

**Comment by MattB70** [comment]
This crash happens on Forge 40.2.0 when Oculus, Iris, FancyMenu, and most importantly DrippyMenu are installed. The crash happens as soon as the game window opens, probably while trying to initialize the loading screen. Disabling Drippy solves it for now.

**Comment by Keksuccino** [comment]
Does it happen everytime you try to start the game?

**Comment by Keksuccino** [comment]
If so, could you please try it with this build and send me the `logs/latest.log` file after? (please send the log even if the game is not crashing)
https://drive.google.com/file/d/1XVBLFk2BGDg6A-UaiBuE1sE2X9q5DzXG/view?usp=share_link

**Comment by MattB70** [comment]
### Your 2.1.2 build:
If the version of Drippy is 2.1.2 the game launches but the loading screen is the default loading screen rather than my drippy loading screen. I suppose this may be due to me using an older loading screen I created last year.

2.1.2 log: https://gist.github.com/MattB70/c9b34b1d500856e88c8746cf49bf3ff2

### Some previous version hopping:
If the version of Drippy is [2.0.0, 2.1.1] the game launches okay, but the loading screen is the default loading screen rather than my drippy loading screen. I suppose this may be due to me using an older loading screen I created last year.

If the version of Drippy is (,1.6.6] the game launches okay, and my custom loading screen is shown.

### Possible source of the crash:
This time while testing I experienced no crashes. The only change I've made since my crash was replacing my FancyMenu animated background with an MP4 using the video extension. Perhaps it's related to the animation on the main menu?

**Comment by MattB70** [comment]
I replaced the mp4 background with the animation again, and experienced no crashes. So that's not conclusive. I'll report back with logs and mod versions as soon as I have another crash. I'll use this 2.1.2 version and re-make my loading screen so I'm up to date with everything.

**Comment by Keksuccino** [comment]
Thank you for testing!

Well yeah, the reason for the default-looking loading screen is switching from Drippy v1 to Drippy v2.

So the game is working fine now? Did you experience crashes everytime you tried to start the game with Oculus/Iris and Drippy before this update or did it just happen once?

**Comment by MattB70** [comment]
The crashing happened a few times in a row before I posted, and I assumed it was a constant crash, but now I'm running everything as I was before but with the provided version of Drippy with no issues. So it seems to be working for now. 

It's possible the provided version of Drippy helped fix it, but the crashing did also stop before I used it as well. Really hard to say what caused it.

I've updated my loading screen to working with Drippy V2 and everything seems to be working good. If I have another crash similar to the first I'll let you know.
